docs/whats_new/version_0_20_0.rst

```diff
@@ -0,0 +1,8 @@
+Version 0.20.0
```

Comment:
I think we have more features like the process connector ?!

Reply:
I added process connector and optionnal id for basic test